k8s-deploy/pkg/job/job.go

```diff
@@ -26,196 +26,187 @@ import (
+// 				}
+// 				dbErr = job.SetStatus(modules.JobStatusCanceled)
+// 				if dbErr != nil {
+// 					log.Error().Err(dbErr).Msg("job.SetStatus error")
+// 				}
+// 				continue
+// 			}
+
+// 			if stopJob(job, cluster) {
+// 				continue
+// 			}
+
+// 			if job.TimeOut() {
+// 				dbErr := job.SetResult("Checkout cluster status timeOut!")
+// 				if dbErr != nil {
+// 					log.Error().Err(dbErr).Msg("job.SetResult error")
+// 				}
+// 				dbErr = job.SetStatus(modules.JobStatusFailed)
+// 				if dbErr != nil {
+// 					log.Error().Err(dbErr).Msg("job.SetStatus error")
+// 				}
+// 				break
+// 			}
+
+// 			// update subJobs
+// 			ClusterStatus, err := service.GetClusterPodStatus(clusterArgs.Name, clusterArgs.Namespace)
+// 			if err != nil {
+// 				log.Error().Err(err).Msg("GetClusterPodStatus error")
+// 			}
+
+// 			log.Debug().Interface("ClusterStatus", ClusterStatus).Msg("GetClusterPodStatus()")
+// 			subJobs := generateSubJobs(job, ClusterStatus)
+
+// 			dbErr = job.SetSubJobs(subJobs)
+// 			if dbErr != nil {
+// 				log.Error().Err(dbErr).Msg("job.SetSubJobs error")
+// 			}
+
+// 			if service.CheckClusterStatus(ClusterStatus) {
+// 				dbErr := job.SetStatus(modules.JobStatusSuccess)
+// 				if dbErr != nil {
+// 					log.Error().Err(dbErr).Msg("job.SetStatus error")
+// 				}
+// 				break
+// 			}
+// 			time.Sleep(1 * time.Second)
+// 		}
+
+// 		if job.Status == modules.JobStatusCanceled {
+// 			job.Result = "Job canceled"
+// 		}
+
+// 		// save cluster to modules
+// 		if job.Status == modules.JobStatusSuccess {
+// 			cluster.Status = modules.ClusterStatusRunning
+// 			cluster.Revision += 1
+// 			_, err = cluster.UpdateByUuid(job.ClusterId)
+// 			if err != nil {
+// 				log.Error().Err(err).Interface("cluster", cluster).Msg("update cluster error")
+// 			}
+// 			log.Debug().Str("chart_version", cluster.ChartVersion).Str("status", cluster.Status.String()).Str("cluster uuid", cluster.Uuid).Msg("update cluster success")
+// 		}
+
+// 		// rollBACK
+// 		if job.Status != modules.JobStatusSuccess && job.Status != modules.JobStatusCanceled {
+// 			_, err := cluster.Delete()
+// 			if err != nil {
+// 				log.Error().Err(err).Interface("cluster", cluster).Msg("DB delete cluster error")
+// 			}
+// 			err = cluster.HelmDelete()
+// 			if err != nil {
+// 				log.Error().Err(err).Interface("cluster", cluster).Msg("helm delete cluster error")
+// 			}
+
+// 			log.Debug().Str("cluster uuid", cluster.Uuid).Msg("install cluster rollBACK success")
+// 		}
+
+// 		job.EndTime = time.Now()
+// 		_, err = job.UpdateByUuid(job.Uuid)
+// 		if err != nil {
+// 			log.Error().Err(err).Str("jobId", job.Uuid).Msg("update job By Uuid error")
+// 		}
+
+// 		if job.Status == modules.JobStatusSuccess {
+// 			log.Debug().Interface("job", job).Msg("job run success")
+// 		} else {
+// 			log.Warn().Interface("job", job).Msg("job run failed")
+// 		}
+
+// 	}()
+
+// 	return job, nil
+// }
```

Comment:
Any reason to keep this chunk of commented code instead of removing them?

Reply:
I suggest remove all unused codes, cuz we have modern CVS now. ;)